packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -58,3 +58,37 @@ EEWorkflowController.put('/:workflowId/share', async (req: WorkflowRequest.Share
 
 	return res.status(200).send();
 });
+
+EEWorkflowController.get(
+	'/:id',
```

Comment:
you could do `/:id(\\d+)` here, and then remove the check for `new` as well as the `isNaN` check.